regulations/views/partial_search.py

```diff
@@ -35,7 +35,7 @@ def reduce_results(self, results, page):
         accordingly. @TODO this is a hack -- we should be able to limit
         results in the request instead"""
         # API page size is API_PAGE_SIZE, but we show only PAGE_SIZE
-        page_idx = (page % (API_PAGE_SIZE / PAGE_SIZE)) * PAGE_SIZE
+        page_idx = int((page % (API_PAGE_SIZE / PAGE_SIZE)) * PAGE_SIZE)
```

Comment:
This doesn't have quite the same meaning. I think we want

```python
(page % (API_PAGE_SIZE // PAGE_SIZE)) * PAGE_SIZE
```
with `//` for integer division